src/main/scala/nl.knaw.dans.easy.deposit/docs/DatasetMetadata.scala

```diff
@@ -15,52 +15,50 @@
  */
 package nl.knaw.dans.easy.deposit.docs
 
-import nl.knaw.dans.easy.deposit.docs.DatasetMetadata.AccessCategory.AccessCategory
-import nl.knaw.dans.easy.deposit.docs.DatasetMetadata.DateQualifier.DateQualifier
-import nl.knaw.dans.easy.deposit.docs.DatasetMetadata.PrivacySensitiveDataPresent.PrivacySensitiveDataPresent
-import nl.knaw.dans.easy.deposit.docs.DatasetMetadata.RelationQualifier.RelationQualifier
 import nl.knaw.dans.easy.deposit.docs.DatasetMetadata._
-import nl.knaw.dans.easy.deposit.docs.JsonUtil.{ InvalidDocumentException, RichJsonInput, toJson }
-import org.joda.time.DateTime
-import org.joda.time.format.ISODateTimeFormat
+import nl.knaw.dans.easy.deposit.docs.JsonUtil.{ InvalidDocumentException, RichJsonInput }
+import nl.knaw.dans.easy.deposit.docs.dm.Date.dateSubmitted
+import nl.knaw.dans.easy.deposit.docs.dm.DateQualifier.DateQualifier
+import nl.knaw.dans.easy.deposit.docs.dm.PrivacySensitiveDataPresent.PrivacySensitiveDataPresent
+import nl.knaw.dans.easy.deposit.docs.dm._
 import org.json4s.JsonInput
 
 import scala.util.{ Failure, Success, Try }
 
-case class DatasetMetadata(identifiers: Option[Seq[SchemedValue[String]]] = None,
-                           languageOfDescription: Option[SchemedKeyValue[String]] = None,
+/**
+ * Params are (de)serialized by [[JsonUtil]].
+ * Lazy public values remix the params for XML serialization with [[DDM]].
+ */
+case class DatasetMetadata(identifiers: Option[Seq[SchemedValue]] = None,
+                           languageOfDescription: Option[SchemedKeyValue] = None,
                            titles: Option[Seq[String]] = None,
                            alternativeTitles: Option[Seq[String]] = None,
                            descriptions: Option[Seq[String]] = None,
                            creators: Option[Seq[Author]] = None,
                            contributors: Option[Seq[Author]] = None,
-                           audiences: Option[Seq[SchemedKeyValue[String]]] = None,
-                           subjects: Option[Seq[PossiblySchemedKeyValue[String]]] = None,
-                           alternativeIdentifiers: Option[Seq[SchemedValue[String]]] = None,
+                           audiences: Option[Seq[SchemedKeyValue]] = None,
+                           subjects: Option[Seq[PossiblySchemedKeyValue]] = None, //TODO xml
+                           alternativeIdentifiers: Option[Seq[SchemedValue]] = None,
                            relations: Option[Seq[RelationType]] = None,
-                           languagesOfFiles: Option[Seq[PossiblySchemedKeyValue[String]]] = None,
-                           dates: Option[Seq[QualifiedSchemedValue[String, DateQualifier]]] = None,
+                           languagesOfFiles: Option[Seq[PossiblySchemedKeyValue]] = None,
+                           dates: Option[Seq[Date]] = None,
```

Comment:
make this and other fields that require further analysis private

Reply:
causes problems for testing:
https://github.com/DANS-KNAW/easy-deposit-api/blob/3e6b9fb3ea8344c914517b31a49d14daaa2825fc/src/test/scala/nl.knaw.dans.easy.deposit/docs/DDMSpec.scala#L184